packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/library_resilient/TestLibraryResilient.py

```diff
@@ -0,0 +1,88 @@
+# TestLibrray.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2019 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+"""
+Test `@_implementationOnly import` in a resilient library used by the main executable
+"""
+import commands
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import os.path
+import time
+import unittest2
+
+class TestLibraryResilient(TestBase):
+
+    mydir = TestBase.compute_mydir(__file__)
+
+    def launch_info(self):
+        info = lldb.SBLaunchInfo([])
+
+        if self.getPlatform() == "freebsd" or self.getPlatform() == "linux":
+            # LD_LIBRARY_PATH must be set so the shared libraries are found on
+            # startup
+            library_path = os.environ.get("LD_LIBRARY_PATH", "")
+            if library_path:
+                library_path += ":"
+            library_path += self.getBuildDir()
+
+            info.SetEnvironmentEntries(["LD_LIBRARY_PATH=" + library_path], True)
+
+        return info
+
+    @swiftTest
+    @expectedFailureOS(no_match(["macosx"])) # Requires Remote Mirrors support
```

Comment:
@adrian-prantl @dcci Got a better idea for this condition?

Reply:
That condition LGTM.